rfcs/0009_Unified_Static_Memory_Planning.md

```diff
@@ -0,0 +1,467 @@
+    Feature Name: Unified Static Memory Planner
+    Start Date: 2021 June 1
+    RFC PR: #0009
+    GitHub Issue: https://github.com/apache/tvm/issues/8404
+
+# Background
+
+Currently, given a ML model primarily TVM will generate two main artifacts :
+
+* A1 : Description of the sequential execution of operators :
+  1. If the "executor" is "graph", this would be a JSON
+  2. if the "executor" is "aot", this would be a main function describing call graph of operators
```

Comment:
```suggestion
  2. if the "executor" is "aot", this would be a main function describing call graph of operators
  3. if the "executor" is "vm", this would be a series of VM bytecode instructions
```

Reply:
Done